docs/how-to/batch-operations/create.md

```diff
@@ -0,0 +1,270 @@
+
+To run a CREATE task, you will need:
+
+* a **CSV** containing your repository item metadata
+* a **config YAML** file that specifies your task type and all options related to that task
+* a directory of **media files** (please see [this section](!!!) if ingesting metadata records without media)
+
+The Arca Office has prepared templates for the first two items in this list. Visit the [Templates page](/arca-docs/how-to/batch-operations/templates) for more detail on how to download and work with them.
+
+Please note there are different templates for **single objects** (e.g., photographs, documents) and **paged content** (e.g., book pages, newspaper issues). 
+
+As values in the metadata spreadsheet will need to comply with the metadata standards and formats outlined in the Arca [Ingest Guide](/arca-docs/how-to/managing-content/ingest-guide), make sure to familiarize yourself or your staff with the guide before working on a Workbench batch.
+
+!!! warning "CSV Encoding"
+    Your CSV should use UTF-8 encoding. 
+    
+    While the Arca metadata template is provided in `.xlsx` format to allow for data validation on controlled fields, we recommend using LibreOffice or Open Office instead of Excel when saving your final CSV to ensure that it is properly encoded and that special characters render appropriately.
+
+## CREATE Task Steps
+
+### 1. Prepare Media Files
+Create a subdirectory in your `islandora_workbench` directory. This subdirectory will contain files specific to this task. 
+
+We recommend giving the directory a descriptive filename that makes the content within easily identifiable (e.g., `sunflower_photo_archive`). Deposit any media files for this ingest into the project subdirectory.
+
+### 2. Prepare Metadata CSV
+Download the appropriate metadata template for your ingest type (single objects or paged content).
+
+Fill out the metadata on the first sheet. There should be one row for every item you are ingesting. Once you've finished filling out the metadata, confirm that the number of rows matches the number of media files you have.
+
+Ensuring it uses UTF-8 encoding, save the first sheet as a CSV into the same project subdirectory as your media files. Give it a descriptive name (e.g., `sunflower_photo_archive.csv`)
+
+### 3. Prepare Config YAML 
+Download the appropriate config YAML for your ingest type (single objects or page content). Give your YAML template a distinct filename for the current ingest (e.g., `sunflower_photo_archive.yml`).
+
+Replace anything in ==square brackets==, as these are required settings that are likely to change from task to task. Pay particular attention to the directives in the headers that start with `# \\`, as some sections contain settings that should not be altered.
+
+Save your config YAML in your root `islandora_workbench` directory.
+
+### 4. Perform Checks
+Open your preferred **terminal** application. Navigate to your `islandora_workbench` directory:
+
+``` bash
+cd islandora_workbench
```

Comment:
this is more `cd /path/to/islandora_workbench` since we don't know where they'll install it.